src/Quartz/JobBuilder.cs

```diff
@@ -67,9 +67,11 @@ public class JobBuilder : IJobConfigurator
     {
         private JobKey? key;
         private string? description;
-        private Type jobType = null!;
+        private Type? jobType;
         private bool durability;
         private bool shouldRecover;
+        private bool? _concurrentExecutionDisallowed;
```

Comment:
currently we don't have `_` prefix deployed around the code base, I'm fine with using it, just need to make things align